transform/babel-plugin-fbt/bin/translate.js

```diff
@@ -140,7 +142,15 @@ const argv = yargs
   .default(args.PRETTY, false)
   .describe(args.PRETTY, 'pretty print the translation output')
   .describe(args.HELP, 'Display usage message')
-  .alias(args.HELP, 'help').argv;
+  .alias(args.HELP, 'help')
+  .string(args.SPLIT_BY_LOCALE_TO)
+  .default(args.SPLIT_BY_LOCALE_TO, null)
```

Comment:
Assuming that the new argument name will be `OUTPUT_DIR`, let's add an alias for it named `o`.

E.g.

`.alias(args.OUTPUT_DIR, 'o')`

Reply:
Fixed in 27cc3d4